physx/src/convex_mesh.rs

```diff
@@ -18,6 +39,93 @@ impl ConvexMesh {
     pub unsafe fn from_raw(ptr: *mut physx_sys::PxConvexMesh) -> Option<Owner<ConvexMesh>> {
         unsafe { Owner::from_raw(ptr as *mut Self) }
     }
+
+    /// Returns the number of vertices.
+    pub fn get_nb_vertices(&self) -> u32 {
+        unsafe { PxConvexMesh_getNbVertices(self.as_ptr()) }
+    }
+
+    /// Returns the vertices.
+    pub fn get_vertices(&self) -> &[PxVec3] {
+        let vertices = unsafe {
+            std::slice::from_raw_parts(
+                PxConvexMesh_getVertices(self.as_ptr()),
+                self.get_nb_vertices() as usize,
+            )
+        };
+
+        // SAFETY: PxVec3 is repr(transparent) wrapper of physx_sys::PxVec3
+        unsafe { std::mem::transmute(vertices) }
```

Comment:
Couldn't we cast the pointer and construct the slice with the right type? Just fewer things being casted.